GCS/probe0/probe0.go

```diff
@@ -0,0 +1,143 @@
+package probe0
+
+import (
+	"context"
+	"fmt"
+	"sync"
+	"io"
+	"os"
+	"time"
+	"cloud.google.com/go/storage"
+	"github.com/google/cloudprober/logger"
+	"github.com/google/cloudprober/metrics"
+	"github.com/google/cloudprober/probes/options"
+	"github.com/google/cloudprober/targets/endpoint"
+	"github.com/googleinterns/step224-2020/blob/evan/hermesMain/cmd"
+)
+
+// Probe holds aggregate information about all probe runs, per-target.
+type HermesProbe struct {
```

Comment:
Why is this "HermesProbe"? Are there other possible probe types?
Is this type going to be used in all probes in Hermes?